.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -152,7 +152,11 @@ if [ "${KATA_WITH_SYSTEM_QEMU}" == "yes" ]; then
 	enable_sandboxedcontainers_extension
 	# Export additional information for the installer to deal with
 	# the QEMU path, for example.
-	oc apply -f ${deployments_dir}/configmap_installer_qemu.yaml
+	if oc version | grep -q "^Server Version: 4.10"; then
```

Comment:
This looks fragile:

- What happens for v4.11?
- What if `oc` gets i18n'd? You could set `LC_ALL` / `LANG` to protect against that.

Reply:
Hi @jodh-intel , currently only 4.11 and 4.10 is supported but I will include `die` in case of a different version. Also I will use the json output of `oc` to parse the version so to avoid any problem with i18n. Thanks for the recommendations!